typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -372,16 +372,43 @@ class ContextFunder {
   async fund(): Promise<boolean> {
     let failureOccurred = false;
 
+    // Returns whether an error occurred
+    const gracefullyHandleError = async (
```

Comment:
I feel like this should be made a function as it is not specific to `fund` and does not capture any local values.

Reply:
good call